components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -677,6 +677,9 @@
 .error_updating_consents {
   margin-top: 1em;
   margin-left: 2em;
+}
+.view-title-required-not-granted {
+  font-size: 1.15em;
```

Comment:
Whether it looks better or not is subjective. We're all used to having that section in smaller fonts so that may influence the perception a lot. But I think there are too many font sizes on the screen already and having that section in "normal" font can make everything look cleaner. Just my 2 cents. Other opinions are welcome.